src/databases/Blueprint/avtBlueprintFileFormat.C

```diff
@@ -747,6 +744,88 @@ avtBlueprintFileFormat::AddBlueprintMeshAndFieldMetadata(avtDatabaseMetaData *md
     }
 }
 
+// ****************************************************************************
+// helper method used to add expression meta data for a blueprint mesh.
+//
+// Mark C. Miller, Wed May  6 12:26:33 PDT 2020
+// ****************************************************************************
+static void
```

Comment:
I made a static to avoid having to change header file which I thought was a no-no for a patch release